changelog.d/app-2379.fixed

```diff
@@ -0,0 +1,2 @@
+Fixed nondeterministic failure of test_api test (again) by splitting out the
```

Comment:
Probably doesnt need to go into user facing changelog

Reply:
Good call. Removed!